handbook/marketing/content/web-series.md

```diff
@@ -0,0 +1,71 @@
+## Web series
+
+Describe what a web series is and why we do them at Sourcegraph
+
+## Dev Tool Time
+
+Dev Tool Time is a bi-weekly web series livestreamed on Twitch where Sourcegraph engineers interview other devs in the community about their desk set up, favorite dev tools, productivity hacks, and more.
```

Comment:
I'd definitely link to previous examples so people can get a better idea of what it's about (and it's also the first think I'd look for if I'd land on that page as a guest)

For example:

```suggestion
Dev Tool Time is a bi-weekly web series livestreamed on [Twitch](https://www.twitch.tv/sourcegraph) where Sourcegraph engineers interview other devs in the community about their desk set up, favorite dev tools, productivity hacks, and more. Recordings are later available on our [YouTube account](https://www.youtube.com/watch?v=W7DHo90Boa4&list=PL6zLuuRVa1_iDEP4EicZ8972RgyccCRGF).
```

But can be phrased differently, or with both links in a separate paragraph:

```
- Our Twitch account: [twitch.tv/sourcegraph](https://www.twitch.tv/sourcegraph)
- YouTube playlist with all previous episodes: [Dev Tool Time on YouTube](https://www.youtube.com/watch?v=W7DHo90Boa4&list=PL6zLuuRVa1_iDEP4EicZ8972RgyccCRGF)
```